modules/openapi-generator/src/main/resources/JavaJaxRS/spec/RestResourceRoot.mustache

```diff
@@ -0,0 +1,5 @@
+package {{invokerPackage}};
+
+public class RestResourceRoot {
+    public static final String ROOT = "{{{contextPath}}}";
```

Comment:
thanks for the PR. one question from me is why do we need to store this `ROOT` separately in another class?

can you store it in `RestApplication` for example instead?

Reply:
I am using interface only, so that i can provide my own RestApplication class, where i add a bunch of exception mappers and such.

The variable could of course be defined in RestApplication, when that class is generated. In the interface only cases, i think, it would be odd to generate a RestApplication class without the @ApplicationPath annotation to hold this ROOT constant, that i am proposing. Btw perhaps APPLICATION_PATH would be a better name for this constant?

Reply:
> Btw perhaps APPLICATION_PATH would be a better name for this constant?

yes, please use it.

Reply:
when ready, please PM me via slack for review and merged.